templates/confirm_auth.html

```diff
@@ -0,0 +1,17 @@
+<html>
+  <head>
+    <title>Bevestig doorgeven gegevens</title>
+  </head>
+
+  <body>
+    De volgende gegevens worden doorgegeven aan het service center:
+    <table>
+      <tr>
+        <td>Email:</td>
+        <td>bla@example.com</td>
```

Comment:
The attributes are hard coded. Is this WIP?

Reply:
No, that is a conscious choice to keep the plugin simpler. Dynamic attributes are not really needed as those are statically configured in the config as well so they wont ever change.